### PR TITLE
Update runtime version to 46

### PR DESCRIPTION
**Pull request by yakushabb**

### CODE REVIEW

**Comment by ghost** [comment]
@hadess Could you review it.

File: fix_appdata.patch [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+--- a/src/org.gnome.OfficeRunner.appdata.xml.in
++++ b/src/org.gnome.OfficeRunner.appdata.xml.in
+@@ -2,13 +2,12 @@
+ <!-- Copyright 2014 William Moreno Reyes <williamjmorenor@gmail.com>
+      Copyright 2014 Bastien Nocera <hadess@hadess.net> -->
+ 
+-<component type="desktop">
+-  <id type="desktop">org.gnome.OfficeRunner.desktop</id>
++<component type="desktop-application">
++  <id>org.gnome.OfficeRunner</id>
```
Comment (bbhtt) [inline fix_appdata.patch]:
```suggestion
+  <id>org.gnome.OfficeRunner.desktop</id>
```

Please don't change the id like we discussed https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#renaming-id-tag

File: org.gnome.OfficeRunner.json [hunk 2]
```diff
@@ -34,7 +34,11 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/office-runner.git",
-                    "commit": "15193aa95b2707619e4b54f7ff6b1f95b721f477"
+                    "commit": "ad339dde48d69bbaec76e87eec75cc37941fdad9"
```
Comment (bbhtt) [inline org.gnome.OfficeRunner.json]: Why is the commit changed?

Comment (ghost) [reply org.gnome.OfficeRunner.json]: To add some trivial changes like translation. I should be careful to stick on the stable releases.

**Comment by ghost** [comment]
@barthalion could you test it.

**Comment by barthalion** [comment]
bot, build

**Pull request merged**